### ISSUE TITLE
Support for round world borders

### ISSUE THREAD

**Issue by christaylor201** [SuperRonanCraft/BetterRTP#52]
**Plugin Version**:
3.0.3

**Request**:
Add support for round world borders.

**How/Why**:
Servers might use a plugin in order to have a round world border, such as [WorldBorder](https://www.spigotmc.org/resources/80466/) or [ChunkyBorder](https://www.spigotmc.org/resources/84278/), which is our case. However, BetterRTP currently only supports square world borders, rather it picks random coordinates in a square.

It'd be nice if there was an option where you could specify in which shape should be the random coordinates picked; maybe something like `Shape` with `square` and `circle` values?

**Additional Context**:
None.

**Comment by SuperRonanCraft** [comment]
I would like for this feature actually, sounds pretty reasonable and probably very easy to implement. I will give you an update on this soon

**Comment by SuperRonanCraft** [comment]
Just added support for this on 3.0.4, please let me know if you find any issues!
https://www.spigotmc.org/resources/betterrtp.36081/updates

**Comment by christaylor201** [comment]
Thank you for adding this, it will help us a lot!

After some testing, I found only one minor issue – the minimum radius isn't respected. I checked the code and I think you must have forgotten to add the `min` variable to the final radius [here](https://github.com/SuperRonanCraft/BetterRTP/blob/563baee57e7b851748bbf2c2c7a13b7d61eb5e3d/src/main/java/me/SuperRonanCraft/BetterRTP/references/worlds/WorldPlayer.java#L156).

To be on the safe side, I generated 20,000 random locations using the code of the [`generateRound()`](https://github.com/SuperRonanCraft/BetterRTP/blob/563baee57e7b851748bbf2c2c7a13b7d61eb5e3d/src/main/java/me/SuperRonanCraft/BetterRTP/references/worlds/WorldPlayer.java#L152-L163) method between radii of 1,000 and 10,000 blocks, see below.

### Current formula
```java
double r = max * Math.sqrt(new Random().nextDouble());
```
![current-formula](https://user-images.githubusercontent.com/53393218/104596377-34628900-5674-11eb-8f53-2803c0da5582.png)

### Fixed formula
```java
double r = max * Math.sqrt(new Random().nextDouble()) + min;
```
![fixed-formula](https://user-images.githubusercontent.com/53393218/104596384-36c4e300-5674-11eb-8703-fd33f8a842e4.png)

**Comment by SuperRonanCraft** [comment]
I want to revisit this, I am very certain I counter acted this, but I want to make sure

**Comment by SuperRonanCraft** [comment]
nope, you are right, I have to offset the radius, thank you very much for letting me know, I will post a patch later today!

**Comment by SuperRonanCraft** [comment]
Fixed in 3.0.4-1 ;)

**Comment by christaylor201** [comment]
Awesome, thank you!
